catalog/message/shape/dataset-shape.ttl

```diff
@@ -0,0 +1,78 @@
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf:   <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix dcat: <http://www.w3.org/ns/dcat#> .
+@prefix odrl: <http://www.w3.org/ns/odrl/2/> .
+@prefix cred: <https://www.w3.org/2018/credentials#> .
+@prefix sec: <https://w3id.org/security#> .
+@prefix prov: <http://www.w3.org/ns/prov#> .
+
+@prefix dspace: <https://w3id.org/dspace/v0.8/> .
+@prefix dspace_shapes: <https://w3id.org/dspace/shapes/v0.8/> .
+
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+
+dspace_shapes:
+	a owl:Ontology ;
+	sh:declare [
+		sh:prefix "rdf" ;
+		sh:namespace "http://www.w3.org/1999/02/22-rdf-syntax-ns#"^^xsd:anyURI ;
+	] ;
+	sh:declare [
+		sh:prefix "dspace" ;
+		sh:namespace "https://w3id.org/dspace/v0.8/"^^xsd:anyURI ;
+	] ;
+.
+
+dspace_shapes:DatasetShape
+	a sh:NodeShape ;
+	sh:targetClass dcat:Dataset ;
+
+    sh:property [
+        a sh:PropertyShape ;
+        sh:path dcat:distribution ;
+        sh:class dcat:Distribution ;
+        sh:minCount 0;
+        sh:severity sh:Violation ;
+        sh:message "<https://raw.githubusercontent.com/International-Data-Spaces-Association/ids-specification/master/schemas/dataset-shape.ttl> (DatasetShape): A dcat:distribution property must point to a dcat:Distribution."@en ;
+    ] ; 
+
+    sh:property [
+        a sh:PropertyShape ;
+        sh:path dcat:keyword ;
+        sh:datatype xsd:string ;
+        sh:minCount 0;
+        sh:severity sh:Violation ;
+        sh:message "<https://raw.githubusercontent.com/International-Data-Spaces-Association/ids-specification/master/schemas/dataset-shape.ttl> (DatasetShape): A dcat:keyword property must point to a xsd:string field."@en ;
+    ] ;
+
+    sh:property [
+        a sh:PropertyShape ;
+        sh:path dct:description ;
+        sh:datatype xsd:string ;
+        sh:minCount 0;
+        sh:severity sh:Violation ;
+        sh:message "<https://raw.githubusercontent.com/International-Data-Spaces-Association/ids-specification/master/schemas/dataset-shape.ttl> (DatasetShape): A dct:description property must point to a xsd:string field."@en ;
+    ] ;
+
+    sh:property [
+        a sh:PropertyShape ;
+        sh:path dct:title ;
+        sh:datatype xsd:string ;
```

Comment:
Please check whether that could be rdf:langString

Reply:
Seems to be xsd:string only